### DISCUSSION TITLE
Input does not support Decimals

### DISCUSSION THREAD

**Discussion by ycaoT** [twilio-labs/paste#1513] [Ideas] [answered]
The only available type for numbers is number, however it only supports pattern="[0-9]*". Wondering if can add step attribute to support Decimal.

https://www.w3schools.com/tags/att_input_step.asp

**Comment by SiTaggart** [comment]
Thanks @ycaoT, we're discussing it amongst the team right now.

**Comment by richbachman** [comment, marked as answer]
Hi @ycaoT. We're still discussing this as a team. If you're blocked by our current implementation, you could still use Input with a pattern that allows decimals. I've built a quick CodeSandbox showing how it might work: https://codesandbox.io/s/angry-feather-ylsi7?file=/src/index.tsx
